enterprise/frontend/src/embedding-sdk/cli/snippets/nextjs-app-snippets.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Custom app snippets for the Pages Router.
+ */
+function getNextJsCustomAppSnippet({
```

Comment:
shouldn't this function be in `nextjs-pages-snippets` instead?

Reply:
ah, the name of the file is a bit confusing indeed. `app` there was referring to "custom app" rather than "app router"

Reply:
Fixed in [f0eb296](https://github.com/metabase/metabase/pull/50281/commits/f0eb296f2bc4461187060975e83dec7b35b6f1f9)